### PR TITLE
Hs 26

### PR DESCRIPTION
**Pull request by wiktor145**

### CODE REVIEW

File: src/test/bassTranslatorTest.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
+function mod(a, b) {
+    while (a < 0) {
+        a += b
+    }
+    return a % b
+}
+
+function contains(list, obj) {
```
Comment (jaksroka) [inline src/test/bassTranslatorTest.js]: Why cannot use functions contains from Utils.js?

File: src/test/bassTranslatorTest.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
+function mod(a, b) {
```
Comment (jaksroka) [inline src/test/bassTranslatorTest.js]: "Mod" have been already defined in Utils.js why not to use it?

File: src/test/bassTranslatorTest.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+var chordElements = JSON.parse('[{"notesNumbers":[0,2,4],"omit":[],"bassElement":{"bassNote":{"pitch":48,"baseNote":0,"chordComponent":1},"symbols":[{"component":3},{"component":5}]},"primeNote":0},{"notesNumbers":[4,6,8],"omit":[],"bassElement":{"bassNote":{"pitch":43,"baseNote":4,"chordComponent":1},"symbols":[{"component":3},{"component":5}]},"primeNote":4},{"notesNumbers":[4,9,7],"omit":[],"bassElement":{"bassNote":{"pitch":43,"baseNote":4,"chordComponent":5},"symbols":[{"component":6,"alteration":"b"},{"component":4}]},"primeNote":0},{"notesNumbers":[5,10,7],"omit":[],"bassElement":{"bassNote":{"pitch":45,"baseNote":5,"chordComponent":3},"symbols":[{"component":6},{"component":3}]},"primeNote":3},{"notesNumbers":[0,2,4],"omit":[],"bassElement":{"bassNote":{"pitch":48,"baseNote":0,"chordComponent":1},"symbols":[{"component":3},{"component":5}]},"primeNote":0}]')
+var mode = "major"
+var meter = [4, 4]
+var durations = [[1, 2], [1, 2], [1, 2], [1, 4], [1, 4]]
+
+
+handleAlterations(harmonicFunctions, chordElements, mode, meter, durations)
+
+console.log(JSON.stringify(harmonicFunctions))
```
Comment (jaksroka) [inline src/test/bassTranslatorTest.js]: There are available some testing tools - look up into other test classes how to use them and please try to change it.

**Review by jaksroka — CHANGES_REQUESTED** [review]

File: src/test/bassTranslatorTest.js (hunk 2 above)
Comment (wiktor145) [reply src/test/bassTranslatorTest.js]: I'll create proper tests in other branch

File: src/test/bassTranslatorTest.js (hunk 3 above)
Comment (wiktor145) [reply src/test/bassTranslatorTest.js]: okay :D

**Pull request merged**